appveyor.yml

```diff
@@ -0,0 +1,43 @@
+os: unstable
+environment:
+  matrix:
+  - nodejs_version: "6"
+  - nodejs_version: "7"
```

Comment:
We could (and should IMO) save build time by only caring about latest LTS (node v6)